Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -204,3 +204,47 @@ the size of the cluster.
     ceph orch set backend ""
     ceph mgr module disable rook
     ```
+
+## External Clusters
+
+Make use of the NFS client running on the external ceph standalone cluster.
```

Comment:
What is specific to external clusters for NFS? I guess for internal mode, the client will be external to the cluster so we don't have an example of PVC/PV in that case. So in this case, the Rook cluster is the one consuming the NFS with a PVC/PV. Maybe a bit more explanation in this paragraph would be helpful for the user.

Reply:
`Rook cluster is the one consuming the NFS`

Rook is not consuming it, they're directly exposing the external nfs client to the kubernetes application.